aptos/p2p_transfer/index.js

```diff
@@ -0,0 +1,97 @@
+import {
+  Account,
+  Aptos,
+  APTOS_COIN,
+  AptosConfig,
+  Ed25519PrivateKey,
+  parseTypeTag,
+  TypeTagAddress,
+  TypeTagU64,
+} from "@aptos-labs/ts-sdk";
+import { getPrometheusMetricPushPayload, pushPrometheusMetricsToVM, sleepAsync } from './common.js';
+
+const COIN_TRANSFER_LATENCY_METRIC_NAME = "e2e_p2p_txn_latency";
+const COIN_TRANSFER_BUILD_LATENCY_METRIC_NAME = "e2e_p2p_txn_latency_build";
+const COIN_TRANSFER_SUBMIT_LATENCY_METRIC_NAME = "e2e_p2p_txn_latency_sign_and_submit";
+const COIN_TRANSFER_SUCCESS_METRIC_NAME = COIN_TRANSFER_LATENCY_METRIC_NAME + "_success";
+const CHAIN_NAME = process.env.CHAIN_NAME;
+const PING_INTERVAL = process.env.PING_INTERVAL * 1000;
+const CUSTOM_NETWORK = process.env.CUSTOM_NETWORK;
+const INDEXER_URL = process.env.INDEXER_URL;
```

Comment:
nit: can we derive the indexer url from the chain name? (for non-custom public networks)